pyrefinebio/compendia.py

```diff
@@ -101,7 +101,12 @@ def download(self, path, prompt=True):
             prompt (bool): if true, will prompt before downloading files bigger than 1GB
         """
         if not self.computed_file.download_url:
-            raise DownloadError("Compendia", "Download url not found - did you set up and activate a Token?")
+            raise DownloadError(
+                "Compendia",
+                "Download url not found - make sure you have set up and activated your Token. "
+                "You can create and activate a new token using pyrefinebio.Token. "
```

Comment:
Are there parenthesis here after `pyrefinebio.Token` and it's not rendering here?
![Screen Shot 2021-02-24 at 11 48 08 AM](https://user-images.githubusercontent.com/15315514/109034981-2e5dc000-7696-11eb-8753-90998dd739fe.png)

Reply:
I left off the parenthesis because pyrefinebio.Token() will create the token but not activate it.
Maybe I should just say something like:
`"You can create a token using pyrefinebio.Token() and then activate it with agree_to_terms_and_conditions(). "`

Reply:
OH.....I thought this was the wrapper function that did both. 

https://github.com/AlexsLemonade/refinebio-py/issues/27 -> I think we should show them whatever this function is when it is implemented.  Here and wherever else we ask them to create a new token.

Reply:
I'm gonna add a note in #27